Content.Client/Arcade/BlockGameMenu.cs

```diff
@@ -519,24 +519,25 @@ public void SetGameoverInfo(int amount, int? localPlacement, int? globalPlacemen
         {
             var globalPlacementText = globalPlacement == null ? "-" : $"#{globalPlacement}";
             var localPlacementText = localPlacement == null ? "-" : $"#{localPlacement}";
-            _finalScoreLabel.Text = Loc.GetString("Global") + $": {globalPlacementText}\n" + Loc.GetString("Local") + $": {localPlacementText}\n" + Loc.GetString("Points") + $": {amount}";
+            _finalScoreLabel.Text =
+                Loc.GetString($"Global: {globalPlacementText}\nLocal: {localPlacementText}\nPoints: {amount}");
```

Comment:
This isn't correct. You need to do string formatting with the actual overload of `GetString`, otherwise it will try to localize the string with all the numbers filled out which will never match.
```suggestion
                Loc.GetString("Global: {0}\nLocal: {1}\nPoints: {2}", globalPlacementText, localPlacementText, amount);
```